.pipelines/singletenancy/ipam-dropgz/ipam-dropgz-cilium-e2e-job-template.yaml

```diff
@@ -0,0 +1,29 @@
+parameters:
```

Comment:
Copy/pasting like this is going to create additional maintenance overhead, and the designs will immediately begin to drift apart. We should evaluate what is actually different between the two pipelines and parameterize the templates accordingly so that they are generic enough to run in both scenarios.